scripts/buildx.sh

```diff
@@ -11,16 +11,21 @@
 ## ./scripts/buildx.sh --profile emqx --pkgtype zip --arch arm64 --builder ghcr.io/emqx/emqx-builder/4.4-4:24.1.5-3-debian10
 
 set -euo pipefail
+set -x
 
 help() {
     echo
-    echo "-h|--help:           To display this usage information"
-    echo "--profile <PROFILE>: EMQ X profile to build, e.g. emqx, emqx-edge"
-    echo "--pkgtype zip|pkg:   Specify which package to build, zip for .zip and pkg for .rpm or .deb"
-    echo "--arch amd64|arm64:  Target arch to build the EMQ X package for"
-    echo "--src_dir <SRC_DIR>: EMQ X source ode in this dir, default to PWD"
-    echo "--builder <BUILDER>: Builder image to pull"
-    echo "                     E.g. ghcr.io/emqx/emqx-builder/4.4-4:24.1.5-3-debian10"
+    echo "-h|--help:                   To display this usage information"
+    echo "--profile <PROFILE>:         EMQ X profile to build, e.g. emqx, emqx-edge"
+    echo "--pkgtype zip|pkg|elixirpkg: Specify which package to build, zip for .zip,"
```

Comment:
maybe add a `--elixir` boolean option?
instead of changing the `--pkgtype`

Reply:
I added it here because I didn't create the option of making a `zip` for Elixir, only the `pkg` equivalent.
If we want to create zips for Elixir as well, then the flag is better indeed.

Reply:
yes, it's still a great quick-start test option to be able to download, unzip and run.

Reply:
Ok, I added an option to make zips with Elixir.